packages/astro/src/integration/index.ts

```diff
@@ -100,6 +100,16 @@ function findDefaultSdkInitFile(type: 'server' | 'client'): string | undefined {
 }
 
 function getSourcemapsAssetsGlob(config: AstroConfig): string {
+  // The vercel adapter puts the output into its .vercel directory
+  // However, the way this adapter is written, the config.outDir value is update too late for
+  // us to reliably detect it. Also, server files are first temporarily written to <root>/dist and then
+  // only copied over to <root>/.vercel. This seems to happen too late though.
+  // So we glob on both of these directories.
+  // Another case of "it ain't pretty but it works":(
+  if (config.adapter && config.adapter.name?.startsWith('@astrojs/vercel')) {
```

Comment:
```suggestion
  if (config.adapter?.name?.startsWith('@astrojs/vercel')) {
```

Reply:
Whoops, good catch, just saw it too late. Will sneak your suggestion into #9668

Reply:
[0a82e21](https://github.com/getsentry/sentry-javascript/pull/9668/commits/0a82e2188864e514c6e5a321b2ce499285adc79a)